.eleventy.js

```diff
@@ -619,6 +619,19 @@ module.exports = function(eleventyConfig) {
         return nav;
     });
 
+    eleventyConfig.addCollection("publications", function(collectionApi) {
+        return [
+            ...collectionApi.getFilteredByTag("whitepaper").map(item => {
```

Comment:
Adding two collections and doing a single map on te elements would have been nicer IMO